example/FFTConvolutionOverRegionOfInterest.ipynb

```diff
@@ -0,0 +1,665 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Generate an input image and visualize ROI\n",
+    "\n",
+    "\n",
+    "def make_image() -> image_type:\n",
+    "    arr = np.ones([150, 150]).astype(np.float32)\n",
+    "    arr[20:80, 20:80] = 5\n",
+    "    arr[70:120, 60:100] = 0\n",
+    "    arr[30:60,50:120] = 10\n",
+    "    image = itk.image_from_array(arr)\n",
+    "    return image\n",
+    "\n",
+    "\n",
+    "def make_roi() -> region_type:\n",
+    "    region = itk.ImageRegion[2]()\n",
+    "    region.SetSize([80, 50])\n",
+    "    region.SetIndex([30, 50])\n",
```

Comment:
Does this notebook use both numpy ordering (Y, X) and ITK ordering (X, Y) of indices?  If so, for whichever is the minority usage, perhaps add comments to indicate that these lines are different from the majority.  E.g.
```python
p = a[4,5] # Uses numpy index ordering
```

Reply:
`numpy` is used only in constructing the input image, all other indices are in ITK format. I've added a note on index ordering just before input image generation.